src/StatusBarMenu.ts

```diff
@@ -206,31 +202,10 @@ export class StatusBarMenu {
                 return false;
             }
 
-            // Register the event handlers in case we need to update binaries.
```

Comment:
💡 just a thought: Will this cause any issue with folks in another version trying to upgrade to latest?